components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -567,6 +568,7 @@ export class GitpodServerEEImpl<C extends GitpodClient, S extends GitpodServer>
             } else if (prebuiltWorkspace.state === 'queued' || prebuiltWorkspace.state === 'building') {
                 if (mode === CreateWorkspaceMode.ForceNew) {
                     // in force mode we ignore running prebuilds as we want to start a workspace as quickly as we can.
+                    // TODO: Fall back to `prebuiltWorkspace.parentPrebuildId` if it's available (i.e. finished successfully)
```

Comment:
Thought: Maybe don't fall back here, but earlier in the call tree (e.g. for when there is no running prebuild yet -- there we can't check for `prebuiltWorkspace.parentPrebuildId`)